docs/ref/request-response.txt

```diff
@@ -638,6 +638,12 @@ Typical usage is to pass the contents of the page, as a string, to the
     >>> response = HttpResponse("Here's the text of the Web page.")
     >>> response = HttpResponse("Text only, please.", content_type="text/plain")
 
+You can also pass a :class:`bytes` or :class:`memoryview` object to the
+constructor::
+
+    >>> response = HttpResponse(b"Here's the text in bytes.")
+    >>> response = HttpResponse(memoryview(b"Here's a memoryview object."))
+
```

Comment:
How about this @timgraham?

Reply:
Please rebase your branch to pickup the changes from e449c3a832ff2a4e3fa83cec6909d0476ed14110.  A `.. versionchanged:: 3.0` annotation should also be added.